src/fix/main.c

```diff
@@ -68,6 +68,80 @@ static void print_usage(void)
 	exit(1);
 }
 
+static long parse_cartridge(char *arg, char **ep)
+{
+	long cartridge = -1;
+
+	if (!arg || *arg == '\0' || *arg == '+' || *arg == '-' ||
+		(*arg >= '0' && *arg <= '9'))
+		return strtoul(arg, ep, 0);
+
+	*ep = arg + strlen(arg);
+
+	/* cartridge type names from Pan Docs; none of them start with [-+0-9] */
+	/* https://gbdev.io/pandocs/#_0147-cartridge-type */
+	if (!strcmp(arg, "ROM ONLY"))
+		cartridge = 0x00;
```

Comment:
Maybe we should also use an `enum`, then

Reply:
In a larger program I'd agree, but this way each name occurs once (as a string) and each value occurs once (right next to the corresponding name string(s)). An enum would be `ROM_ONLY = 0`, then redundantly `if "ROM ONLY" cartridge = ROM_ONLY`.

Reply:
But maybe an array of string and value?

```
struct {
    const char *name;
    unsigned int value;
} cart_types[] = {
    {"ROM_ONLY", 0x00},
    ...
    {NULL, 0}
};
```

Reply:
Then you just iterate with strcmp.

Reply:
I considered that, but the ad-hoc struct also felt excessive. If that's just me, though, I can take that approach to avoid so many repeated `strcmp`s.

Reply:
The struct could be moved out at some point if it is useful for something else (like printing the type of MBC from the value in the header).

Reply:
Done.